src/config/configuration.rs

```diff
@@ -1,33 +1,29 @@
-use crate::config::grpc_configuration::GrpcConfiguration;
-use crate::config::transport_strategy::TransportStrategy;
 use std::time::Duration;
 
+use crate::config::transport_strategy::TransportStrategy;
+
 /// Configuration for a Momento cache client.
 ///
 /// Static, versioned configurations are provided for different environments:
 /// ```
 /// use momento::config::configurations;
 ///
 /// /// Use laptop for local development
-/// let developer_config = configurations::laptop::LATEST;
+/// let developer_config = configurations::laptop::latest();
 /// /// Use in_region for a typical server environment
-/// let server_config = configurations::in_region::V1;
+/// let server_config = configurations::in_region::v1();
 /// ```
 /// If you have specific requirements, configurations can also be constructed manually:
 /// ```
 /// use std::time::Duration;
 /// use momento::config::configuration::Configuration;
-/// use momento::config::grpc_configuration::GrpcConfiguration;
+/// use momento::config::grpc_configuration::GrpcConfigurationBuilder;
 /// use momento::config::transport_strategy::TransportStrategy;
 ///
 /// let config = Configuration {
```

Comment:
I think that if we're going to show an example of this here, we need to do it via `Configuration::builder().blah` as well